semgrep_output_v1.atd

```diff
@@ -931,6 +931,10 @@ type historical_info = {
    * secrets which are buried in the git history which we wouldn't find at HEAD
    *)
   git_commit: sha1;
+  (* Git blob at which the finding is present. Sent in addition to the commit
+   * since some SCMs have permalinks which use the blob sha, so this information
+   * is useful when generating links back to the SCM. *)
+  git_blob: sha1;
```

Comment:
Should we also keep git_blob as optional to make it backwards compatible?

Reply:
Yeah, I was just writing a comment about this. The record in which this currently appears is not yet used, so there isn't a real compatibility concern and I am skeptical the added complexity of making this optional at the usages is better than not.

Reply:
Ahh, gotcha. If this is part of a record that is unused, then LGTM.